.editorconfig

```diff
@@ -11,12 +11,12 @@ max_line_length = 80
 trim_trailing_whitespace = true
 
 [*.{md,yaml,yml}]
-indent_style = space
 indent_size = 2
 
 [*.py]
-indent_style = space
-indent_size = 4
+max_line_length = 88
+
+[*.talon]
 max_line_length = 88
```

Comment:
Maybe we should just set `max_line_length` to 88 globally, under `[*]`?

Reply:
Should we keep 80 for `[*.{md,yaml,yml}]`, or just use 88 everywhere for simplicity?

Reply:
I'd say everywhere for simplicity.